### PR TITLE
Emily/weeklydebriefbutton

### PR DESCRIPTION
**Pull request by emilyhu8**
added a button visually

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -54,13 +54,14 @@ repositories {
...
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-
+    implementation "io.grpc:grpc-okhttp:1.32.2"
     implementation 'androidx.appcompat:appcompat:1.3.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.1'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    implementation "androidx.core:core-ktx:1.7.0"
+    implementation "androidx.c" +
```
Comment (BenHarris4848) [inline app/build.gradle:63]: Accidental newline here

**Review by BenHarris4848 — CHANGES_REQUESTED** [review]
Sorry for the delay on this review. One small comment but otherwise good job

File: app/src/main/java/com/cornellappdev/volume/adapters/MorePublicationsAdapter.kt [hunk 2]
```diff
@@ -25,12 +25,16 @@ class MorePublicationsAdapter(
...
     private val mAdapterOnClickHandler: AdapterOnClickHandler?,
-    private val isOnboarding: Boolean = false) :
+    private val isOnboarding: Boolean = false,
+    private val mAdapterOnClicker:AdapterOnClicker?) :
         RecyclerView.Adapter<MorePublicationsAdapter.MorePublicationVH>() {
 
     interface AdapterOnClickHandler {
         fun onFollowClick(wasFollowed: Boolean)
     }
+    interface AdapterOnClicker{
```
Comment (dishanest) [inline app/src/main/java/com/cornellappdev/volume/adapters/MorePublicationsAdapter.kt:35]: insert newline before

File: app/src/main/java/com/cornellappdev/volume/adapters/MorePublicationsAdapter.kt [hunk 3]
```diff
@@ -133,16 +137,7 @@ class MorePublicationsAdapter(
...
-            intent.putExtra(Publication.INTENT_KEY, currentItem)
-            intent.putParcelableExtra(
-                NavigationSource.INTENT_KEY, if (isOnboarding) {
-                    NavigationSource.ONBOARDING
-                } else {
-                    NavigationSource.MORE_PUBLICATIONS
-                }
-            )
-            view.context.startActivity(intent)
+            mAdapterOnClicker?.onMorePublicationClicked(currentItem,isOnboarding)
```
Comment (dishanest) [inline app/src/main/java/com/cornellappdev/volume/adapters/MorePublicationsAdapter.kt:140]: add space between parameters

File: app/src/main/java/com/cornellappdev/volume/fragments/PublicationsFragment.kt [hunk 4]
```diff
@@ -319,7 +319,8 @@ class PublicationsFragment : Fragment(), FollowingHorizontalAdapter.AdapterOnCli
                             MorePublicationsAdapter(
                                 morePublications,
                                 prefUtils,
-                                this@PublicationsFragment
+                                this@PublicationsFragment,
+                                    mAdapterOnClicker = this@PublicationsFragment
```
Comment (dishanest) [inline app/src/main/java/com/cornellappdev/volume/fragments/PublicationsFragment.kt:323]: indentation?

File: app/src/main/java/com/cornellappdev/volume/fragments/PublicationsFragment.kt [hunk 5]
```diff
@@ -373,4 +374,19 @@ class PublicationsFragment : Fragment(), FollowingHorizontalAdapter.AdapterOnCli
...
+    override fun onMorePublicationClicked(publication: Publication, isOnboarding: Boolean) {
+        val intent = Intent(view?.context, PublicationProfileActivity::class.java)
+        intent.putExtra(Publication.INTENT_KEY, publication)
+        intent.putParcelableExtra(
+                NavigationSource.INTENT_KEY, if (isOnboarding) {
+            NavigationSource.ONBOARDING
+        } else {
+            NavigationSource.MORE_PUBLICATIONS
+        }
+        )
```
Comment (dishanest) [inline app/src/main/java/com/cornellappdev/volume/fragments/PublicationsFragment.kt:388]: Indentation. There should be an Android linter somewhere, would be good to ask @aasthashah999

File: app/src/main/res/layout/fragment_home.xml [hunk 6]
```diff
@@ -20,13 +20,14 @@
...
                 android:layout_marginLeft="6dp"
-                android:layout_marginTop="80dp"
+                android:layout_marginTop="20dp"
                 android:contentDescription="@string/description_main_volume_logo"
                 app:layout_constraintLeft_toLeftOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
                 app:srcCompat="@drawable/volume_title"
                 tools:srcCompat="@drawable/volume_title" />
 
+
```
Comment (dishanest) [inline app/src/main/res/layout/fragment_home.xml:30]: remove extra newlines in this file

**Review by dishanest — APPROVED** [review]
Lookin good! A few nits here and there.

**Review by BenHarris4848 — APPROVED** [review]
Looks good!

**Pull request merged**